components/serverless/Makefile

```diff
@@ -34,21 +34,20 @@ test-local:
 	@go tool cover -func=/tmp/artifacts/cover.out | grep total | awk '{print $$3}'
 
 # Generate manifests e.g. CRD, RBAC etc.
-CONTROLLER_GEN_VERSION=v0.6.2
-CONFIGOPERATOR = config/operator
```

Comment:
I removed it because it was corrupted.
As we have plans to adjust makefiles in repo I want to keep it withouth any big changes till the refactor.